<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="ioFrameLayout">
      <item>
       <widget class="QLabel" name="ioLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Input/Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="inputFileFrame">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="inputPathEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Input File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="inputPathButton">
           <property name="accessibleName">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	border-radius: 10px;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="outputPathFrame">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="outputPathEdit">
           <property name="placeholderText">
            <string>Output Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="outputPathButton">
           <property name="styleSheet">
            <string notr="true">QToolButton {
	border-radius: 10px;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
